src/containers/AdminTeamEditor/index.jsx

```diff
@@ -64,7 +64,9 @@ class AdminTeamEditor extends Component {
       "assignmentPriority"
     ]);
     if ("assignmentPriority" in team) {
-      team.assignmentPriority = parseInt(team.assignmentPriority, 10);
+      const rawPriority = team.assignmentPriority;
+      team.assignmentPriority =
+        rawPriority === "" ? 500 : parseInt(team.assignmentPriority, 10);
```

Comment:
nitpick: parseInt for `rawPriority` since we're already reading that value.

Reply:
Thanks, whole reason I wrote that 1st line and still forgot to change that lol